packages/squidex/.eslintrc.js

```diff
@@ -3,5 +3,6 @@ module.exports = {
   rules: {
     'lines-between-class-members': 'off',
     'no-unused-vars': 'off',
+    'max-classes-per-file': 'off',
```

Comment:
In the future, we should enable this feature.
And for the number of lines of code :P